src/tab/header/item/_tab-header-item.scss

```diff
@@ -0,0 +1,25 @@
+.tab-header-item {
+  --active-color: 50, 50, 211;
```

Comment:
Should we make this variable name more specific (maybe `tab-header-active-color`) ? It sounds too generic so it may conflict with other variable names